Documentation/direct-tools.md

```diff
@@ -1,113 +0,0 @@
----
```

Comment:
Why are you removing this file? You should probably have a separate commit for it unless that's related to the main change.

Reply:
I found this written in the issue
`This really shouldn't be used anymore and we can remove this topic`
Can you please explain it a little further?